include/regs.h

```diff
@@ -49,6 +49,7 @@
 #define R_ENV_FOG_NEAR                           REG(14)
 #define R_ENV_TIME_SPEED_OLD                     REG(15) // Most likely used during development. Unused in the final game.
 #define R_RUN_SPEED_LIMIT                        REG(45)
+#define R_DECELERATE_RATE                        REG(43)
```

Comment:
```suggestion
#define R_DECELERATE_RATE                        REG(43)
#define R_RUN_SPEED_LIMIT                        REG(45)
```
43 comes before 45 :)

Reply:
thanks